lib/axios.js

```diff
@@ -27,11 +27,14 @@ function getLabels(exit, token, owner, host, repository) {
         })
         .catch(function (err) {
             // If response.status is 404
-            if (err.response.status == 404) {
-                echo.error('404: Not found.')
-                echo.error('URL seems to be faulty: ' + url)
-                echo.error('Please check your arguments and try again.', true)
-            } else {
+            if (err.response) {
```

Comment:
Yeah. It was rare, so I never actually saw what the real error was, but I think it might have been a rate limit error after essentially spamming the api